packages/uui-tabs/lib/uui-tab.element.ts

```diff
@@ -120,11 +141,28 @@ export class UUITabElement extends ActiveMixin(LabelMixin('', LitElement)) {
   }
 
   render() {
-    return html`
-      <button type="button" ?disabled=${this.disabled} role="tab">
-        ${this.renderLabel()}
-      </button>
-    `;
+    return this.href
+      ? html`
+          <a
+            id="button"
+            href=${ifDefined(!this.disabled ? this.href : undefined)}
```

Comment:
This component should probably use the same logic for the `rel` attribute as you just implemented in uui-button, which is missing here.

Reply:
Yep, done now :-)